asyncnsq/tcp/connection.py

```diff
@@ -196,7 +196,7 @@ def _upgrade_to_deflate(self):
     async def _read_data(self):
         """Response reader task."""
         is_canceled = False
-        logger.info("{} starting _read_data".format(self))
+        # logger.debug("{} starting _read_data".format(self))
```

Comment:
debug message if only for you to debug you program, it is not a noise log message

Reply:
I added this message in debugging, and then removed it in the next commit to clean it up.